File: API_Demo/API_Demo/Models/Task.cs
using System;
using System.Collections.Generic;

namespace API_Demo.Models;

public partial class Task
{
    public int Id { get; set; }

    public int? Idparent { get; set; }

    public string? Label { get; set; }

    public string? Type { get; set; }

    public string? Name { get; set; }

    public DateTime? StartDate { get; set; }

    public DateTime? EndDate { get; set; }

    public int? Duration { get; set; }

    public float? Progress { get; set; }

    public bool? IsUnscheduled { get; set; }
}

File: Demo/src/app/task/models/task.ts
export interface TaskModelMap {
	id: number;
	start_date: string;
	duration: number;
	text: string;
	progress: number;
	end_date: string;
	parent: number;
}

export interface Task {
	id: number;
	idparent: number;
	label: string;
	type: string;
	name: string;
	startDate: string;
	endDate: string;
	duration: number;
	progress: number;
	isUnscheduled: boolean;
}
